blocks/color-palette/test/index.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { ColorPalette } from '../';
+
+describe( 'ColorPalette', () => {
+	const colors = [ 'red', 'white', 'blue' ];
+	const currentColor = 'red';
+	const onChange = jest.fn();
+
+	const wrapper = shallow( <ColorPalette colors={ colors } value={ currentColor } onChange={ onChange } /> );
```

Comment:
I can see why you're doing all these initializations outside the `test` calls but an issue with this way of writing the unit tests is that a `test` could alter these global variables and break the following tests. I'd suggest trying to keep each test as isolated as possible.

One way to do it to reinstantiate these variables in `beforeEach`

Reply:
To clarify, change these to let and then rebind them in `beforeEach()` ?

Reply:
yep, I also don't mind repeating these small instantiations from test to test.

Reply:
> yep, I also don't mind repeating these small instantiations from test to test.

I am fine with that as well, but most of the tests in this project don't do that, and everyone else seems to be more in favor of this approach of declaring them at the top and reusing a single declaration. Should we open an issue to figure out what we want to do to be consistent?

Reply:
Not worth an issue IMO but feel free to do so if you think so. :) Just pick the option you like

Reply:
> Just pick the option you like

I like consistency 😄